web/src/utilities/repeater.js

```diff
@@ -0,0 +1,28 @@
+export class Repeater {
+    constructor(handlerFunc, sleepMs) {
+        this.handlerFunc = handlerFunc;
+        this.sleepMs = sleepMs;
+        this.stop = false;
+    }
+
+    Start = () => {
```

Comment:
this can start with small letter like the other functions

Reply:
@salahalsaleh1993 but what's the convention?  I made small case on the ones that shouldn't be called directly.

Reply:
not entirely sure but i think small letter is the convention